src/USER-CONP/compute_conp_matrix.cpp

```diff
@@ -266,44 +245,19 @@ void ComputeConpMatrix::init() {
 
 void ComputeConpMatrix::setup() {
   igroupnum = group->count(igroup);
-  jgroupnum = group->count(jgroup);
-  ngroup = igroupnum + jgroupnum;
+  ngroup = igroupnum;
```

Comment:
Warum machst du das so? Du könntest doch direkt igroupnum in ngroup umbennen. igroupnum brauchen wir ja eigentlich nicht mehr oder?